thingsboard.yaml

```diff
@@ -105,8 +105,11 @@ subpackages:
     description: "Core service responsible for handling REST API, WebSocket subscriptions, and processing messages via the rule engine."
     dependencies:
       runtime:
+        - fontconfig
+        - lcms2
         - openjdk-17
```

Comment:
Remove and just keep the JVM for this and the other subpackage

Reply:
When removing fontconfig and lcms from the runtime dependencies the program errors out.

```Caused by: java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-17-openjdk/lib/liblcms.so: liblcms2.so.2: cannot open shared object file: No such file or directory```

Reply:
I was just talking about the JDK. Other deos can stay :)